gocognit.go

```diff
@@ -57,6 +58,15 @@ func recvString(recv ast.Expr) string {
 		return t.Name
 	case *ast.StarExpr:
 		return "*" + recvString(t.X)
+	case *ast.IndexExpr:
+		return recvString(t.X) + "[" + recvString(t.Index) + "]"
+	case *ast.IndexListExpr:
+		targs := make([]string, 0, len(t.Indices))
+		for _, exp := range t.Indices {
+			targs = append(targs, recvString(exp))
```

Comment:
The `targs` slice was created using `make` with the non-zero length. Why do we still use `append`?

Reply:
I am using the second version of `make` for creating slices, where the second argument is the length of the slice and the third argument is the capacity of the underlying array. In this case go won't resize the underlying array, because I specified enough capacity. https://go.dev/play/p/rZmxsFAvmF1.

I can change it to:
```go
targs := make([]string, len(t.Indices))
for i, exp := range t.Indices {
	targs[i] = recvString(exp)
}
```